test/widgets/content_test.dart

```diff
@@ -290,6 +297,79 @@ void main() {
 
   testContentSmoke(ContentExample.mathBlock);
 
+  testContentSmoke(ContentExample.strong);
+
+  testContentSmoke(ContentExample.emphasis);
+
+  group('InlineCodeNode', () {
+    testContentSmoke(ContentExample.inlineCode);
+
+    testWidgets('maintains font-size ratio with surrounding text', (tester) async {
+      /// Simulate a nested span's style by merging all ancestor-span styles,
+      /// starting from the root.
+      ///
+      /// This isn't how the style actually gets applied in the code under test,
+      /// but it should hopefully simulate it closely enough.
+      TextStyle? mergeStylesOuterToInner(InlineSpan outerSpan, InlineSpan innerSpan) {
+        final styles = <TextStyle?>[];
+        bool recurse(InlineSpan span) {
+          styles.add(span.style);
+          if (span == innerSpan) {
```

Comment:
```suggestion
          if (identical(span, innerSpan)) {
```

I.e. the equivalent of JS `===`.

Otherwise this calls the `operator ==` override, which can accept more things as equal.